ui/admin/app/templates/scopes/scope/targets/target/injected-application-credential-sources.hbs

```diff
@@ -0,0 +1,146 @@
+{{page-title
+  (t 'resources.target.injected-application-credential-source.title_plural')
+}}
+
+<Rose::Layout::Page as |page|>
+
+  <page.breadcrumbs>
+    <BreadCrumbs />
+  </page.breadcrumbs>
+
+  <page.header>
+    <Targets::Target::Header @model={{@model}} />
+  </page.header>
+
+  <page.navigation>
+    <Targets::Target::Nav @model={{@model}} />
+  </page.navigation>
+
+  <page.actions>
+    <Targets::Target::Actions @model={{@model}} />
+  </page.actions>
+
+  <page.body>
+    {{#if @model.injectedApplicationCredentialSources}}
+
+      <Rose::Table as |table|>
+        <table.header as |header|>
+          <header.row as |row|>
+            <row.headerCell>{{t 'form.name.label'}}</row.headerCell>
+            <row.headerCell>{{t 'form.type.label'}}</row.headerCell>
+            <row.headerCell>{{t 'form.id.label'}}</row.headerCell>
+            <row.headerCell>{{t 'titles.actions'}}</row.headerCell>
+          </header.row>
+        </table.header>
+        <table.body as |body|>
+          {{#each
+            @model.injectedApplicationCredentialSources
+            as |injectedApplicationCredentialSource|
+          }}
+            <body.row as |row|>
+              <row.headerCell>
+                {{#if
+                  (eq
+                    injectedApplicationCredentialSource.type 'username_password'
```

Comment:
Is there a property on the model to determine the type (e.g. isUsernamePassword)? I feel like we've been using a property for some models like `isSsh`

Reply:
here the model is `target`, and its types are `TCP and SSH`. `username_password ` is a credential type and we can't use the credential model property here.

Reply:
Are these not credentials models? Is the issue that it could be different type of models or are these just completely different objects? I thought we would be getting models back from this [property](https://github.com/hashicorp/boundary-ui/blob/609295c776b2716853c5bf77cd56e7fdf26c776a/addons/api/addon/models/target.js#L81)

Reply:
`injectedApplicationCredentialSources` doesn't return credentials or cred lib models. It updates the `injected_application_credential_source_ids` which Is an array field on the target model.

Reply:
You are right, this [array](https://github.com/hashicorp/boundary-ui/blob/609295c776b2716853c5bf77cd56e7fdf26c776a/addons/api/addon/models/target.js#L81) contains the credential models which are associated with the target. So I can add isUsernamePassword property to the cred model and should have access within that target